netmap/types.proto

```diff
@@ -121,7 +121,39 @@ message NodeInfo {
   // Ways to connect to a node
   string address = 2;
 
-  // Attributes of the NeoFS node.
+  // Administrator-defined Attributes of the NeoFS Storage Node.
+  //
+  // Node's attributes are mostly used during Storage Policy evaluation to
+  // calculate object's placement and find a set of nodes satisfying policy
+  // requirements. There are some "well-known" node attributes common to all the
+  // Storage Nodes in the network and used implicitly with default values if not
+  // explicitly set:
+  //
+  // * Capacity \
+  //   Total available disk space in Gigabytes.
+  // * Price \
+  //   Price in GAS tokens for storing one GB of data during one Epoch.
```

Comment:
Just a short note: this could be a small fraction of a number, but network map operates with `uint64` price values. We have to be careful with it because it is not the value that will be directly stored in network map.